paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -727,6 +746,7 @@ class PaparazziPluginTest {
     if (!gradleProperties.exists()) {
       gradleProperties.createNewFile()
       gradleProperties.writeText("android.useAndroidX=true")
+      gradleProperties.writeText("android.nonTransitiveRClass=true")
```

Comment:
Initially I added a `gradle.properties` to the test project, but figured there wasn't any harm in just adding it here. Let me know which you prefer.

Reply:
This will add that property to every test, vs having one test that highlights the specific change.

Reply:
Alrighty. I'll add the `gradle.properties` to the individual project. 👍